types/uid-generator/tsconfig.json

```
@@ -4,6 +4,7 @@
         "lib": [
             "es6"
         ],
+        "target":            "ES2015",
```

```suggestion
        "target": "ES2015",
```
Why so many whitespaces here? xD

---

This can't be true.
I have only committed this Remove-Space modification, then DefinitelyTyped.BenchmarkPR fail?